mocha_test/autoInject.js

```diff
@@ -74,4 +74,17 @@ describe('autoInject', function () {
         });
     });
 
+    // Needs to be run on ES6 only - not sure how you plan to handle this
```

Comment:
I'd be hesitant to add this until we have a good test for it.  It would be nice to do some sort of feature detection and conditionally run this test, but an unsupported arrow function will cause a syntax error before the test even gets evaluated.  Using Babel won't work because it will compile it to a `function () {}` and not really test the feature.   We need a strategy for this.